package.json

```diff
@@ -40,14 +39,15 @@
     "ember-export-application-global": "^1.0.5",
     "ember-frost-core": "^1.14.3",
     "ember-frost-test": "^1.0.0",
-    "ember-load-initializers": "^0.5.1",
-    "ember-prop-types": "3.11.0",
+    "ember-load-initializers": "^0.6.0",
+    "ember-prop-types": "^3.11.0",
```

Comment:
You changed the blueprint from `^3.11.0` to `3.11.0`, but changed _package.json_ from `3.11.0` to `^3.11.0`.